torch/onnx/__init__.py

```diff
@@ -140,6 +141,11 @@ def symbolic_override_packed_sequence_based(symbolic_fn):
 
     def might_trace(args):
         import torch
-        return torch._C._jit_is_tracing(args[0][0])
+        first_arg = args[0]
+        if not isinstance(first_arg, torch.nn.utils.rnn.PackedSequence):
+            raise ValueError('pad_packed_sequence expects sequence to be a '
+                             'PackedSequence, but got an object of type {}'
+                             .format(type(first_arg)))
+        return torch.is_tensor(first_arg[0]) and torch._C._jit_is_tracing(first_arg[0])
```

Comment:
This error message looks nice, let's add something like it above. Also, I think the `is_tensor` check is redundant here, as `.data` in `PackedSequence` is always a tensor

Reply:
Btw, should we be using `isinstance(first_arg[0], torch.Tensor)` ?

Reply:
We should. I forgot about it